package.json

```diff
@@ -23,6 +23,7 @@
   "dependencies": {
     "dotenv": "^16.0.0",
     "express": "^4.18.2",
+    "jest-sorted": "^1.0.14",
```

Comment:
jest sorted and supertest can go in your `devDependencies`, prime for pruning on deploy